### ISSUE TITLE
listMultipartUploads does not work with S3Client (v2)

### ISSUE THREAD

**Issue by jerry-cherry** [adobe/S3Mock#734]
S3Client.listMultipartUploads() doesn't seem to hit `FileStoreController.listMultipartUploads()`. It works for V1 client and S3Client (v2) works against AWS S3.

It seems to be related to the request URL mapping:
- `software.amazon.awssdk.services.s3.transform.ListMultipartUploadsRequestMarshaller` binds the path as `/{Bucket}?uploads`
- `FileStoreController` binds it as `/{bucketName:.+}/`

Here is the test code.

```
    @ClassRule
    public static S3MockRule S3_MOCK_RULE = S3MockRule.builder().silent().build();

        try (var s3Client = S3_MOCK_RULE.createS3ClientV2()) {
            s3Client.createBucket(builder -> builder.bucket(bucketName));

            s3Client.createMultipartUpload(
                builder -> builder.bucket(bucketName).key(keyPath));

            var response = s3Client.listMultipartUploads(builder -> builder.bucket(bucketName));
            assertEquals(1, response .uploads().size());

            for (var upload : response.uploads()) {
                s3Client.abortMultipartUpload(builder -> builder.bucket(bucketName).key(batchPath).uploadId(upload.uploadId()));
            }

            s3Client.deleteBucket(builder -> builder.bucket(bucketName));
        }
```

**Comment by jerry-cherry** [comment]
Thanks Arne.

**Comment by afranken** [comment]
just released 2.4.16 including this fix.
Should be available in the next few minutes / hours on Docker hub and in the Maven one repo. :)

**Comment by jerry-cherry** [comment]
sweet!!!
